cmd/createcluster.go

```diff
@@ -521,3 +581,66 @@ func aggSign(secrets [][]*bls_sig.SecretKeyShare, message []byte) ([]byte, error
 
 	return b, nil
 }
+
+// loadDefinition returns the cluster definition from disk or an HTTP URL. It also verifies signatures
+// and hashes before returning the definition.
+func loadDefinition(ctx context.Context, defFile string) (cluster.Definition, error) {
+	var def cluster.Definition
```

Comment:
why define this if you return early below?

Reply:
I updated it now so that definition downloaded from both flows, ie, disk and network, are verified before returned